test/test_process_pdos.py

```diff
@@ -0,0 +1,42 @@
+
+import numpy as np
+import galore
+import unittest
+from numpy.testing import assert_array_almost_equal
+
+
+
+class test_process_pdos(unittest.TestCase):
+
+    def test_lorentzian(self):
```

Comment:
test_lorentzian and test_broaden don't test `process_pdos` so I don't think they belong in this class?

Reply:
Still, they appear to be new tests and it's good to test them _somewhere_. Really the test suite could use a bit of modernisation (Pytest maybe?) So for now let's keep them.